src/components/modal/modal.js

```diff
@@ -719,7 +720,7 @@ export const BModal = /*#__PURE__*/ Vue.extend({
         !content ||
         document === target ||
         contains(content, target) ||
-        this.ignoreEnforceFocusSelectors.some(selector => closest(selector, target, true))
+        this.ignoreEnforceFocusSelectors.some(selector => !!closest(selector, target, true))
```

Comment:
`closest` will work with a string of selectors separated by `,` (the comma is like an "or"), and would be faster than iterating over the array with some.

```js
contains(content, target) ||
(this.ignoreEnforceFocusSelectors && closest(this.ignoreEnforceFocusSelectors, target, true)
```

Would require the computed prop to return a string (which would be an empty string if no selectors specified)

```js
ignoreEnforceFocusSelectors() {
  return concat(this.ignoreEnforceFocusSelector).filter(identity).join(',').trim()
}
```